airbyte-ci/connectors/pipelines/pipelines/dagger/actions/python/common.py

```diff
@@ -159,12 +159,13 @@ def _install_python_dependencies_from_poetry(
 ) -> Container:
     pip_install_poetry_cmd = ["pip", "install", "poetry"]
     poetry_disable_virtual_env_cmd = ["poetry", "config", "virtualenvs.create", "false"]
```

Comment:
Does this imply that inside of the connector images, poetry will install deps into global python package space?

Reply:
I think this allows us to then run the connector with the default entry point, as opposed to having to do `poetry run whatever.py`, right?

Reply:
> Does this imply that inside of the connector images, poetry will install deps into global python package space?

Yes, building the docker image is a two step approach:
* We provision a builder container on which we run the package installation: this is where poetry is a requirement
* We mount the installed packages to a new container which is transformed into our final image

This means that `poetry` could eventually not be bundled in our final images, the build dependencies being required only in the builder container.

But for simplicity the builder and the final connector image use the same base image: our python connector base image which has poetry.

If we'd want to optimize the image size we could just install build dependencies in the builder container.